### ISSUE TITLE
Removal request of aresmanga.net from the tachiyomi app

### ISSUE THREAD

**Issue by Volver555** [tachiyomiorg/tachiyomi-extensions#15396]
### Source link

https://aresmanga.net/

### Other details

We changed Domains from aresmanga.com to aresmanga.net, 
so here we are once again requesting a removal of our source,
thank you very much for your help!

Best regards

### Requirements

- [X] Proof of ownership/intent to remove sent to a Tachiyomi Discord server mod via DM
- [X] Site only hosts content scanlated by the group and not stolen from other scanlators or official releases (i.e., not an aggregator site)
- [X] Site is not infested with user-hostile features (e.g., invasive or malicious ads)

**Comment by BLACKxSUN** [comment]
They are steal from other source
Look at this work all chapters under 154 Belonging to other source like team x , ozul scane , swatmanga 

https://aresmanga.net/series/the-eternal-supreme/
This pic from 154 , even the logo for ozul scane showed

![Screenshot_٢٠٢٣٠٣٠٦-٠٣١٠٠٧_Tachiyomi](https://user-images.githubusercontent.com/93000786/222994333-aec8449a-fa6c-43e5-9d90-5323f452f7cd.jpg)

Not only this work , but alot of them under date 2023/1/1

**Comment by BLACKxSUN** [comment]
@CarlosEsco  if you can cheak this
Thanks

**Comment by Volver555** [comment]
Alright so I'll try to make this as brief as I can...

These are contents that has been found on our website and claimed to not be ours:

"I'm the guy who found contents from other groups on your website and added the extension back"
https://aresmanga.net/martial-peak-chapter-1/ - Cover says Baha Elden
https://aresmanga.net/martial-god-asura-chapter-1/ - MangaSwat watermark
https://aresmanga.net/demonic-emperor-chapter-1/ - Team TssE watermark
https://aresmanga.net/martial-arts-reigns-chapter-1/ - SwatManga Watermark
https://aresmanga.net/tales-of-demons-and-gods-chapter-168-5/ Orta Team watermark 
"This are some more examples of content not scanned by your group"

and Here I'll try and Explain every single case of them, we were part of other scanlation groups but mainly team X

firstly there is no baha elden it's a guy who we did in fact get permission from when we started the work that's for 1
secondly, Martial god asura doesn't even exist on MangaSwat cuz we as ares agreed with them to give the serie, kind of a trade with another
thirdly, Demonic emperor was started by a Friend called midou who is also part of our current team (ares)
martial arts reigns was a collaboration work between us and swat manga lately but after like chaps 400 and so we stopped collaboratng to not delay releases, you can see your self that the chaps are the same with different water marks on both sites to prove the collab (kinda like how flame and asura collb'd in solo leveling)
tales of demons and gods is ours as well, the Orta Team became part of ares after they stopped being a blog site thingy and so they kept working on their series 

these events happend like a year ago
One more thing, is that the flame ar also became part of us lately in like b4 3 months
even tho we didn't take any chapters but we have the right to and I can prove it with a message from the guy in flame scans eng who gave us permission
just like the rest
Tld:
Ah, I almsot forgot, I talked to the directory of our team (FL Eng) and they said that it's okey for you to take the chapters and the series...
and if you need any soart of help tell me.
![image](https://user-images.githubusercontent.com/113272610/223103268-2d60bead-64f1-4863-83db-32de1b97f4a7.png)
 
 
 we do have a lot of collab works right now, some of them aren't continuous due to some problems but overall our relation with most of the arabic team is stable, we don't take what's not ours and we try to get permission either by paying a sum to the previous translators that did the chapter or by just asking them if they are kind to give it!
 
 if any doubts persist this is my discord (𝓥𝓸𝓵𝓿𝓮𝓻#4711), I'm confident that I can clear all your doubts with the screens that I have!!!

**Comment by shipoxx** [comment]
> Alright so I'll try to make this as brief as I can...
> 
> These are contents that has been found on our website and claimed to not be ours:
> 
> "I'm the guy who found contents from other groups on your website and added the extension back"
> https://aresmanga.net/martial-peak-chapter-1/ - Cover says Baha Elden
> https://aresmanga.net/martial-god-asura-chapter-1/ - MangaSwat watermark
> https://aresmanga.net/demonic-emperor-chapter-1/ - Team TssE watermark
> https://aresmanga.net/martial-arts-reigns-chapter-1/ - SwatManga Watermark
> https://aresmanga.net/tales-of-demons-and-gods-chapter-168-5/ Orta Team watermark 
> "This are some more examples of content not scanned by your group"
> 
> and Here I'll try and Explain every single case of them, we were part of other scanlation groups but mainly team X
> 
> firstly there is no baha elden it's a guy who we did in fact get permission from when we started the work that's for 1
> secondly, Martial god asura doesn't even exist on MangaSwat cuz we as ares agreed with them to give the serie, kind of a trade with another
> thirdly, Demonic emperor was started by a Friend called midou who is also part of our current team (ares)
> martial arts reigns was a collaboration work between us and swat manga lately but after like chaps 400 and so we stopped collaboratng to not delay releases, you can see your self that the chaps are the same with different water marks on both sites to prove the collab (kinda like how flame and asura collb'd in solo leveling)
> tales of demons and gods is ours as well, the Orta Team became part of ares after they stopped being a blog site thingy and so they kept working on their series 
> 
> these events happend like a year ago
> One more thing, is that the flame ar also became part of us lately in like b4 3 months
> even tho we didn't take any chapters but we have the right to and I can prove it with a message from the guy in flame scans eng who gave us permission
> just like the rest
> Tld:
> Ah, I almsot forgot, I talked to the directory of our team (FL Eng) and they said that it's okey for you to take the chapters and the series...
> and if you need any soart of help tell me.
> ![image](https://user-images.githubusercontent.com/113272610/223103268-2d60bead-64f1-4863-83db-32de1b97f4a7.png)
>  
>  
>  we do have a lot of collab works right now, some of them aren't continuous due to some problems but overall our relation with most of the arabic team is stable, we don't take what's not ours and we try to get permission either by paying a sum to the previous translators that did the chapter or by just asking them if they are kind to give it!
>  
>  if any doubts persist this is my discord (𝓥𝓸𝓵𝓿𝓮𝓻#4711), I'm confident that I can clear all your doubts with the screens that I have!!!

Still some content is from your team as example "Apotheosis"
Unless if you translate 1k+ chapter by your team then it's still "stealing" last time i checked one of leaders in ur time literally told me he took some chapters lol

**Comment by Volver555** [comment]
@shipoxx  igth so i'm on my way to college.. but for you to claim something you gotta have some soart of proof,  you don't even know that our team was and still run by 1 and only 1 leader which is me so this guy or whatever you talked to probably just some random so unless you provide me with something true (role or something on our server) what you're saying isn't irrelevant, also if you wanna talk about apotheosis precisely it's one of zeus's series go talk to him on our discord server, he will explain it to you, about how to we got permission and so on!

![image](https://user-images.githubusercontent.com/113272610/223110074-94ce81ce-6c8d-42bb-89be-db42d36e6fb4.png)

**Comment by shipoxx** [comment]
> @shipoxx  igth so i'm on my way to college.. but for you to claim something you gotta have some soart of proof,  you don't even know that our team was and still run by 1 and only 1 leader which is me so this guy or whatever you talked to probably just some random so unless you provide me with something true (role or something on our server) what you're saying isn't irrelevant, also if you wanna talk about apotheosis precisely it's one of zeus's series go talk to him on our discord server, he will explain it to you, about how to we got permission and so on!
> 
> ![image](https://user-images.githubusercontent.com/113272610/223110074-94ce81ce-6c8d-42bb-89be-db42d36e6fb4.png)
> 
> 

As i said last time i checked, i don't have much time but this need to be reviewed carefully

سواء سوات او تيم أو فراق اخرى كلها اخير مرة تكلمت مع قادة قالوا لي انكم "تسرقون أعمالهم"
وهذه ليست اول مرة لا يزال لدي سكرينشوت في حاسوبي

**Comment by arkon** [comment]
Reopening for discussion purposes.

**Comment by BLACKxSUN** [comment]
If you want your site to be original, translate all the chapters with your own effort without stealing "some" to complete the work, otherwise all sites that do this are stolen and you have no right to remove it from the application.

**Comment by osamaelshaeth** [comment]
> إذا كنت تريد أن تقوم بذلك ، فإنه يجب أن تقوم بذلك ، أو تقوم بإدراجها في الموقع.

Before u make any hostile response to the party u are talking to, you can carefully check what he wanted to write first of all, everyone knows what Arabic Manha translation sites are and what they do, everyone takes from the other and you can see this in all sites, that any site starts working from when the other site ended, and if it cannot compete, it deletes the work or takes the 'previous and current' chapters and starts removing its watermarks and starts changing some words until it notices Others The difference is that there is no theft, so the second party does not have the right to claim, from my point of view I see that sites that leave the names of the owners of the work do something good compared to others.

**Comment by Tatsumi25293** [comment]
I have been following this site for a long time, and I know very well that it is not a thief, but rather it only completes the works that were left by other sites, so it does not modify the watermarks or other things that other sites do.

**Comment by Volver555** [comment]
@BLACKxSUN 
Alright... it seems to me that you got no clue whatsoever about what you are saying, you're straight up spitting nonesens at this point, you also seem to me that you didn't even read what I wrote before nor did you provide any proof, you claim something different everytime, so lemme clear you in:

Aresmanga isn't a new website we've been here for around 1 year and a half or something, starting with iimanga... you don't think we're a bunch of randoms who came to the domain, we were in fact part of other teams in before, like teamx, azora, swat, andromeda... 

if we take you're claims into consideration, you claim that us taking series with permission of their previous TLs/TSs is considered theft? and even us putting the chapters that we did in our previous websites when we used to work there is considered theft as well? 

alright ares manga's team is composed with more than 120 members in total, while I cannot prove this for any of them, I can prove myself at least, imma give you a serie that I did in every website I worked on, and you can imagine the rest of the team and as a disclaimer, I alone has done over 3000 chapters in the last 4 years!

https://team1x1.fun/series/villain-to-kill/1

let's take a look at this chapter, it clearly states my name, and guess what do you see it in aresmanga? No... Can I still get it to our website? well, in you're statement I have no right to do so... you don't expect me, I the one who worked for all these series for free to leave them as they are?

-------------------------------------------------------------
![image](https://user-images.githubusercontent.com/113272610/223287580-e8df0f1d-3475-4ff0-a157-1c41e71899c6.png)

this screen from 2021 proves me working with azora on the serie called now (Return of the 8th Class Magicien), and btw if you're wondering why brahimm is speaking in english? well he is the same guy from asura, he is a tunisian friend of mine... and I hope this clear azora out of the way
-------------------------------------------------------------
Concerning ozul most of our series with them are collabs, while I don't have screens about every single serie we're working on, I would invite you to do ask em yourself

So to sum this up, we complete series: 
- that we did work on ourselves
- that we took permission from their previous owners
- that are collab'd with other websites or were!


if this doesn't clear your doubts even after I provided you with screenshots, that you're just trolling here and you want the extension back cuz you are with all my due respect another follower that doesn't support the hardworking website!

and I hope you're read the whole thing this time!
